block/src/Pos/Block/Genesis.hs

```diff
@@ -14,6 +14,3 @@ import           Pos.Lrc.Genesis (genesisLeaders)
 ----------------------------------------------------------------------------
 -- Genesis
 ----------------------------------------------------------------------------
```

Comment:
You move a function from the file, but you do it without respect, you don't even check whether the file makes sense afterwards.

Reply:
Jokes aside, why would you do this? It seems that `Pos.Block.Genesis` is a perfect place to put a function that produces the 0th genesis block.

Reply:
Creating a module for just one function, when it can be in other module, is stupid. More modules => harder to understand the code structure. Check out `tree` output after this refactoring:
```
ξ> tree block/src/Pos/Block/
block/src/Pos/Block/
├── Base.hs
├── BHelpers.hs
├── BListener.hs
├── BlockWorkMode.hs
├── Configuration.hs
├── Error.hs
├── Logic
│   ├── Creation.hs
│   ├── Header.hs
│   ├── Integrity.hs
│   ├── Internal.hs
│   ├── Util.hs
│   └── VAR.hs
├── Logic.hs
├── Network
│   ├── #Announce.hs#
│   ├── Logic.hs
│   ├── Retrieval.hs
│   └── Types.hs
├── Network.hs
├── RetrievalQueue.hs
├── Slog
│   ├── Context.hs
│   ├── Logic.hs
│   └── Types.hs
├── Slog.hs
├── Types.hs
└── Worker.hs
```

Reply:
OK, good point.